lib/mini_profiler_rails/railtie.rb

```diff
@@ -30,6 +30,7 @@ def self.initialize!(app)
 
     if serves_static_assets?(app)
       c.skip_paths << app.config.assets.prefix
+      c.skip_paths << "/packs"
```

Comment:
This is a rather silly solution for the problem..
A better solution here would be to find out if the app is using webpacker or not and add this if yes.

Probably, and hopefully, there is something within webpacker that would give me that "/packs" path without me needing to hardcode here... 

HELP!

Reply:
Would `if defined?(Webpacker)` work?

Reply:
Would help indeed.

A simple solution with that could be:

```ruby
c.skip_paths << "/packs" if defined?(Webpacker)
```

Problem is: You can tell Webpacker to put those anywhere else, like this:

![image](https://user-images.githubusercontent.com/773898/76557712-f31dad80-6493-11ea-935b-f9492d366ede.png)

Webpacker allows me to get that path with:

```ruby
Webpacker.config.public_output_path # => #<Pathname:/app/public/packs/like/this>
```

So, I am thinking this needs to be a bit smarter and this is where I start to find my solution ugly:

```ruby
if defined?(Webpacker)
  public_packs_folder = 
    Webpacker.config.public_output_path.to_s.gsub!("#{app.config.root}/public", "")
  c.skip_paths << public_packs_folder
end
```

Is there a cleaner way to do that "substring"?